content/en/docs/concepts/architecture/nodes.md

```diff
@@ -327,15 +327,14 @@ If you want to explicitly reserve resources for non-Pod processes, see
 If you have enabled the `TopologyManager`
 [feature gate](/docs/reference/command-line-tools-reference/feature-gates/), then
 the kubelet can use topology hints when making resource assignment decisions.
-See [Control Topology Management Policies on a Node](/docs/tasks/administer-cluster/topology-manager/)
+See [Control Topology Management Policies on a node](/docs/tasks/administer-cluster/topology-manager/)
```

Comment:
What do you think about reverting this change?
Since this is anchor text and all words are capitalized, leave Node capitalized?